Rosyk_Task4.py

```diff
@@ -0,0 +1,25 @@
+# Multiparadigm programming languages, Task 2
+# Rosyk Nikita, №15
+
+print('Multiparadigm programming languages, Task 2')
+print('Nikita Rosyk gr. IKM-221k, №15')
+
```

Comment:
I'm not sure but suppose you need to leave 2 blank lines before and after a function